rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -41,7 +41,7 @@ class SE3 {
         Returns:
             Array, shape (1,3), representing the point in the w frame.
         */
-        var ones = nj.ones([1, 1]);
+        var ones = nj.ones([1]);
```

Comment:
curious what the shape was before and after, and why it worked before?

Reply:
The variable ones actually continues to represent the same data ([1]) as before. However, this implementation changes the shape from [1, 1] into [1]. The previous implementation was throwing an error on the next line when calling `nj.concatenate` (`ValueError: all the input arrays must have same number of dimensions`).

To address your second question about why this was working before - when we run the JS application, these errors are ignored (i.e. the app might not show what we want but it will not break). The only way to check if this works is with the `se3.test.js` unittest which never passed (link to original [PR](https://github.com/borglab/gtsfm/actions/runs/1271046518/jobs/1552762259) failing the unittest). Also for some reason not all JS tests are run by github (see the screenshot attached it runs only 3 out of 8 tests available, not running the se3.test.js)

![Screenshot from 2022-09-27 20-15-26](https://user-images.githubusercontent.com/54330853/192659468-270fd85a-54e6-417f-aef8-f5601e341050.png)